TrafficCapture/build.gradle

```diff
@@ -39,7 +36,6 @@ subprojects {
 
 allprojects {
     apply plugin: 'java'
-    apply plugin: 'org.owasp.dependencycheck'
```

Comment:
Have we lost anything with this going away?

Reply:
I didn't see anything in CI, other task dependencies, and in documentation.  It looks like this is effectively another CVE scanner and its duplicated with Dependabot's scanner & SonarQube instance.